diff --git a/node_modules/@testing-library/react-native/build/queries/text.js b/node_modules/@testing-library/react-native/build/queries/text.js
index 426dc5a..250a093 100644
--- a/node_modules/@testing-library/react-native/build/queries/text.js
+++ b/node_modules/@testing-library/react-native/build/queries/text.js
@@ -28,12 +28,12 @@ const getChildrenAsText = children => {
       // has no text. In such situations, react-test-renderer will traverse down
       // this tree in a separate call and run this query again. As a result, the
       // query will match the deepest text node that matches requested text.
-      if ((0, _filterNodeByType.filterNodeByType)(child, _reactNative.Text)) {
-        return;
-      }
-      if ((0, _filterNodeByType.filterNodeByType)(child, React.Fragment)) {
+      // if ((0, _filterNodeByType.filterNodeByType)(child, _reactNative.Text)) {
+      //   return;
+      // }
+      // if ((0, _filterNodeByType.filterNodeByType)(child, React.Fragment)) {
         textContent.push(...getChildrenAsText(child.props.children));
-      }
+      // }
     }
   });
   return textContent;
